### PR TITLE
WORK-209:

### PR DESCRIPTION
**Pull request by hsuchie4**
- Allow multiple assets drag and drop into collections

### CODE REVIEW

File: src/tactic/ui/panel/tile_layout_wdg.py [hunk 1]
```diff
@@ -1709,39 +1709,70 @@ def get_scale_wdg(my):
...
-                    parent_code: parent_code,
-                    search_code: src_code
-                };
-                server.insert(collection_type, data);
-                spt.notify.show_message("Added to Collection");
-                if (!dst_top.hasClass("spt_collection_item")){
-                    spt.table.refresh_rows([dst_top], null, null);
-                }
+            // Regular single drag and drop
+            if (selected_tiles.length == 0) {
```
Comment (oboreo) [inline src/tactic/ui/panel/tile_layout_wdg.py]: should be 1

File: src/tactic/ui/panel/tile_layout_wdg.py [hunk 2]
```diff
@@ -1755,9 +1786,8 @@ def get_scale_wdg(my):
             }
         }
     }
-
-    var tile_top = bvr.src_el.getParent(".spt_tile_top");
-    spt.table.refresh_rows([tile_top], null, null);
+    
+    spt.table.refresh_rows([src_top], null, {show_select: true});
```
Comment (oboreo) [inline src/tactic/ui/panel/tile_layout_wdg.py]: change it back to null

File: src/tactic/ui/panel/tile_layout_wdg.py [hunk 3]
```diff
@@ -1709,39 +1709,70 @@ def get_scale_wdg(my):
...
-                }
+            // Regular single drag and drop
+            if (selected_tiles.length == 0) {
+                var src_code = src_top.getAttribute("spt_search_code");
                 
+                if (parent_code != src_code){
+                    var data = {
+                        parent_code: parent_code,
+                        search_code: src_code
+                    };
```
Comment (oboreo) [inline src/tactic/ui/panel/tile_layout_wdg.py]: try {

File: src/tactic/ui/panel/tile_layout_wdg.py [hunk 4]
```diff
@@ -1709,39 +1709,70 @@ def get_scale_wdg(my):
...
+    var bg = row.getElement(".spt_tile_bg");
+    checkbox.checked = true;
+    spt.table.select_row(row);
+    bg.setStyle("opacity", "0.7");
 
     var dst_el = spt.get_event_target(evt);
     var dst_top = dst_el.hasClass("spt_tile_top") ? dst_el : dst_el.getParent(".spt_tile_top");
+    
+    var layout = bvr.src_el.getParent(".spt_layout");
+    var src_top = bvr.src_el.getParent(".spt_tile_top");
```
Comment (oboreo) [inline src/tactic/ui/panel/tile_layout_wdg.py]: var has_error = false;

File: src/tactic/ui/panel/tile_layout_wdg.py [hunk 5]
```diff
@@ -1709,39 +1709,70 @@ def get_scale_wdg(my):
...
+        var parent_code = dst_top.getAttribute("spt_search_code");
 
-            var layout = bvr.src_el.getParent(".spt_layout");
-            var collection_type = layout.getAttribute("spt_collection_type");
-            var src_top = bvr.src_el.getParent(".spt_tile_top");
-            var src_code = src_top.getAttribute("spt_search_code");
-            var parent_code = dst_top.getAttribute("spt_search_code");
+        var collection_type = layout.getAttribute("spt_collection_type");
+
+        if (parent._is_collection == true) {
```
Comment (oboreo) [inline src/tactic/ui/panel/tile_layout_wdg.py]: add a message if user adds to a non collection..

File: src/tactic/ui/panel/tile_layout_wdg.py (hunk 4 above)
Comment (hsuchie4) [inline src/tactic/ui/panel/tile_layout_wdg.py]: fixed

File: src/tactic/ui/panel/tile_layout_wdg.py (hunk 5 above)
Comment (hsuchie4) [inline src/tactic/ui/panel/tile_layout_wdg.py]: fixed

File: src/tactic/ui/panel/tile_layout_wdg.py (hunk 1 above)
Comment (hsuchie4) [inline src/tactic/ui/panel/tile_layout_wdg.py]: fixed

**Pull request merged**